docs/rest.md

```diff
@@ -278,12 +278,12 @@ And when fetching for a list of users (`/users`), the following response is expe
 ]
 ```
 
-If the structure of your API responses differs than what Apollo expects, you can define a `responseTransformer` in the client. This function receives the JSON response as the 1st argument, and the current `typeName` as the 2nd argument.
+If the structure of your API responses differs than what Apollo expects, you can define a `responseTransformer` in the client. This function receives the response object as the 1st argument, and the current `typeName` as the 2nd argument. It should return a `Promise` as it is responsible for reading the response body by calling the appropriate method (e.g. `response.json()`).
 
 ```js
 const link = new RestLink({
   uri: '/api',
-  responseTransformer: response => response.data,
+  responseTransformer: async response => response.json().then(({data}) => data),
```

Comment:
Perhaps we should show a non-standard transform, since this is already the default behavior, it looks like we are telling them they need to do this.

Reply:
Anything in mind? I started to write on with text(), but thought the parsing code seemed verbose and not very relevant.

Reply:
Btw it was my understanding that they actually _would_ need to do this, since the contents of `data` are expected at the top level, but yes it's quite subtle especially with json() in there.